packages/api/src/codegen/languages/typescript/index.ts

```diff
@@ -265,24 +260,19 @@ export default class TSGenerator extends CodeGenerator {
         defaultImport: 'type { ConfigOptions, FetchResponse, HTTPMethodRange }',
         moduleSpecifier: '@readme/api-core',
       },
-      { defaultImport: 'Oas', moduleSpecifier: 'oas' },
       { defaultImport: 'APICore', moduleSpecifier: '@readme/api-core' },
       { defaultImport: 'definition', moduleSpecifier: this.specPath },
     ]);
 
     // @todo add TOS, License, info.* to a docblock at the top of the SDK.
     this.sdk = sourceFile.addClass({
       name: 'SDK',
-      properties: [
-        { name: 'spec', type: 'Oas' },
-        { name: 'core', type: 'APICore' },
-      ],
+      properties: [{ name: 'core', type: 'APICore' }],
```

Comment:
This creates the `spec: Oas` and `core: APICore` properties in the main SDK class.